lib/generators/capistrano/unicorn_nginx/templates/nginx_conf.erb

```diff
@@ -1,18 +1,18 @@
 upstream unicorn_<%= fetch(:nginx_config_name) %> {
-<% if fetch(:unicorn_use_tcp) -%>
-  <% roles(:app).each do |role| -%>
+  <% if fetch(:unicorn_use_tcp) -%>
```

Comment:
Please undo changes to indentation in this file, unless you have a good reason for these